Sources/Danger/GitHubDSL.swift

```diff
@@ -28,7 +29,8 @@ public struct GitHub: Decodable, Equatable {
     public let reviews: [GitHubReview]
 
     public let requestedReviewers: GitHubRequestedReviewers
-
+    
+    internal(set) public var api: Octokit!
```

Comment:
internal? ( maybe I don't get how access rights work here)

Reply:
Just the set is internal (the api is set from DangerDSL, then it can not be `private(set)`)